corehq/apps/app_manager/views/utils.py

```diff
@@ -346,7 +346,9 @@ def update_linked_app(app, user_id):
         report_map = get_static_report_mapping(latest_master_build.domain, app['domain'])
 
         try:
-            app = overwrite_app(app, latest_master_build, report_map)
+            new_version = app.version + 1 if app.version else 1
+            app = overwrite_app(app, latest_master_build, report_map, version=new_version)
+            app.pulled_from_master_version = latest_master_build.version
```

Comment:
nit:out of scope of this PR but a small change,
`latest_master_build` should be `latest_master_releaed_build`

Reply:
you mean renaming the var? `latest_master_build` is already the latest released master build in reality

Reply:
ya i just meant the var name.

Reply:
87a33b077d8